### PR TITLE
add logits error to troubleshooting

### PR DESCRIPTION
**Pull request by mike-ferguson**
Adds the question and answer covering the error:

` assert len(logits['neuroid']) == 1000`

from the console log.

### CODE REVIEW

File: benchmarks/templates/benchmarks/tutorials/tutorial_troubleshooting.html [hunk 1]
```diff
@@ -143,6 +143,26 @@ <h3 class="benefits_heading is-size-3-mobile">How to Fix Common Errors</h3>
...
+            <span class="is-italic">Description</span>: When checking your console (build) log for what might have gone
+            wrong, you see an error akin to this:
+        </p>
+        <pre class="modified_1 collapsable"><code>
+assert len(logits['neuroid']) == 1000
+AssertionError
+        </code></pre>
+        <p class="benefits_info is-size-5-mobile shorter no_top">
+            <span class="is-italic">Cause:</span> This normally arises when Brain-Score tries to score your model on
+            engineering benchmarks. These benchmarks expect a model to be able to do 1000-way ImageNet classification.
```
Comment (mschrimpf) [inline benchmarks/templates/benchmarks/tutorials/tutorial_troubleshooting.html:160]:
```suggestion
            ImageNet engineering benchmarks which often expect a model to be able to do 1000-way classification on ImageNet labels.
```

**Review by mschrimpf — APPROVED** [review]
